tests/Table/TableTest.php

```
@@ -0,0 +1,1125 @@
+                ->tfoot(Tfoot::tag()->row('Total', '1'))
+                ->render(),
+            'Failed asserting that complete table composes correctly using convenience methods.',
+        );
+    }
+
+    public function testRenderWithGlobalDefaultsAreApplied(): void
+    {
+        SimpleFactory::setDefaults(
+            Table::class,
+            ['class' => 'default-class'],
+        );
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\UIAwesome\Html\Core\Factory\SimpleFactory' in method 'testRenderWithGlobalDefaultsAreApplied'. (undefined)

(StaticAccess)